stock_picking_send_by_mail/__manifest__.py

```diff
@@ -5,7 +5,7 @@
 {
     "name": "Stock Picking by Mail",
     "summary": "Send stock picking by email",
-    "version": "15.0.1.0.1",
+    "version": "16.0.1.0.1",
```

Comment:
The version should be `16.0.1.0.0` and must be changed in the migration commit, not in the pre-commit stuff.

Reply:
could you please check it now?